packages/gestalt/src/Controller.js

```diff
@@ -5,14 +5,15 @@ import Contents from './Contents.js';
 import OutsideEventBehavior from './behaviors/OutsideEventBehavior.js';
 
 type Props = {|
-  anchor: ?HTMLElement,
+  anchor: HTMLElement,
   bgColor: 'blue' | 'darkGray' | 'orange' | 'white',
+  caret?: boolean,
   children?: React.Node,
   idealDirection?: 'up' | 'right' | 'down' | 'left',
   onDismiss: () => void,
   positionRelativeToAnchor: boolean,
   shouldFocus?: boolean,
-  size?: 'xs' | 'sm' | 'md' | 'lg' | 'xl' | number,
+  size?: 'xs' | 'sm' | 'md' | 'lg' | 'xl' | number | null,
```

Comment:
Size should be non-optional if you're specifying a `defaultProps`.

Reply:
Good catch, changed